docs/file-formats/colors.md

```diff
@@ -2,26 +2,123 @@
 
 This file defines the design system's colors.
 
-### Specification
+## Specification
 
-At the top level, the file is an object containing a `"colors"` array.
+At the top level, the file is an object containing:
+
+- a `theme` array
+- a `colors` array
+
+### `theme`
+
+The `theme` array is optional. The default theme is the first item of the array, if any.
+
+Each theme has the following attributes:
+
+| Attribute  | Type     | Required | Description                                                                                                                                                                                                                                                                                                            |
+| ---------- | -------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`       | `string` | Yes      | The unique `id` of the theme, which will be used internally by Lona Studio and in the generated code. This should be _code-friendly_: it must not contain spaces or unusual characters, since it will be used as a variable name. It should also be strictly longer than 1 character and cannot be equal to `extends`. |
```

Comment:
Because it would conflict with how you declare the color value:
```js
{
  themes: [{ id: 'r' }],
  colors: [{
    id: 'color',
    value: { r: 0, g: 0, b: 0 } } // does `r` reference the theme or `red`?
  ]
}
```